test/model/avatar_url_test.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/model/avatar_url.dart';
+
+class ExampleUser {
+  final Uri realm;
+  final int userId;
+  final String email;
+
+  ExampleUser({
+    required this.realm,
+    required this.userId,
+    required this.email,
+  });
+}
+
+void main() {
+  var egUser = ExampleUser(
+    realm: Uri.parse("https://zulip.example.org"),
+    userId: 1417,
+    email: "sample@zulip.org",
+  );
```

Comment:
Better to use an actual `User` object. See `example_data.dart` for ways to conveniently make those for tests. See existing tests that import that file for examples to follow of the style for using it.

Reply:
[df73a95](https://github.com/zulip/zulip-flutter/pull/534/commits/df73a950bda05a9d97388b757c2019a4c1d1e276)